script.js

```diff
@@ -1,12 +1,333 @@
 //You can edit ALL of the code here
-function setup() {
-  const allEpisodes = getAllEpisodes();
-  makePageForEpisodes(allEpisodes);
+let allEpisodes; //= getAllEpisodes();
+let allShows; //= getAllShows();
+const rootElem = document.getElementById("root");
+rootElem.setAttribute("class", "root");
+let containerEl = document.getElementById("container");
+
+// all shows
+
+const loadShowList = (showID) => {
+  const urlShows = `https://api.tvmaze.com/shows/${showID}/episodes`;
+  fetch(urlShows)
+    .then((response) => {
+      if (response.status >= 200 && response.status <= 299) {
+        return response.json();
+      } else {
+        throw new Error(
+          `Encountered something unexpected: ${response.status} ${response.statusText}`
+        );
+      }
+    })
+    .then((episodeList) => {
+      allEpisodes = episodeList;
+
+      makePageForEpisodes(allEpisodes);
+    });
+};
+
+function displayAllShows(showList) {
```

Comment:
Great function to populate your home page! It's well commented and clear to follow what happens at each step. To make it even better, think of refactoring it by moving some codes to helper functions so that it doesn't have to deal with everything.